test/conformance/runtime/util.go

```diff
@@ -72,3 +88,17 @@ func runtimeInfo(t *testing.T, clients *test.Clients, names *test.ResourceNames,
 	err = json.Unmarshal(resp.Body, &ri)
 	return names, &ri, err
 }
+
+func splitOpts(opts ...interface{}) ([]ServiceOption, []pkgTest.RequestOption) {
+	serviceOpts := []ServiceOption{}
+	reqOpts := []pkgTest.RequestOption{}
+	for _, opt := range opts {
+		switch opt.(type) {
+		case ServiceOption:
+			serviceOpts = append(serviceOpts, opt.(ServiceOption))
+		case pkgTest.RequestOption:
+			reqOpts = append(reqOpts, opt.(pkgTest.RequestOption))
+		}
```

Comment:
We should fail if we get something else.  Take `*testing.T`?

Reply:
I thought about this, but I didn't see much value as it (1) complicated the logic for handling split (I would prefer returning error over taking testing t) and (2) the function signature suggests that that non-Service and non-Request options would be dropped and that is expected.

Reply:
Alright I fixed it.